src/GitHub/pyproject.toml

```diff
@@ -0,0 +1,43 @@
+[build-system]
+
+requires = [
+    "setuptools>=42",
+    "wheel"
+]
+
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "BL_Python.GitHub"
+requires-python = ">=3.10"
+authors = [
+    {name = 'Aaron Holmes', email = 'aholmes@mednet.ucla.edu'}
+]
+description = 'Libraries for working with the GitHub API in Boutros Lab.'
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "License :: OSI Approved :: GNU General Public License (GPL)",
+    "Operating System :: OS Independent"
+]
+
+dependencies = [
+    "PyGithub~=2.1"
+]
+
+dynamic = ["version", "readme"]
+[tool.setuptools.dynamic]
+version = {attr = "BL_Python.GitHub.__version__"}
+readme = {file = ["README.md"], content-type = "text/markdown"}
+
+[tool.setuptools.package-dir]
+"BL_Python.GitHub" = "BL_Python/GitHub"
+
+[tool.setuptools.packages]
+find = {}
+
+[project.optional-dependencies]
+dev-dependencies = [
+    "pytest == 7.4.0",
+    "pytest-mock == 3.11.1",
+    "mock == 5.1.0"
+]
```

Comment:
Not exactly critical, but it's probably worth adding the newline:

https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

Reply:
Done.